accounts/ethstore/cli/src/main.rs

```diff
@@ -172,7 +172,9 @@ fn key_dir(location: &str, password: Option<Password>) -> Result<Box<dyn KeyDire
 		"geth-test" => RootDiskDirectory::create(dir::geth(true))?,
 		path if path.starts_with("parity") => {
 			let chain = path.split('-').nth(1).unwrap_or("ethereum");
-			let path = dir::parity(chain);
+			let mut path = dir::default_data_path().parse::<std::path::PathBuf>().unwrap();
```

Comment:
we're converting a `PathBuf` in `default_data_path` to a `String` and then parsing it again here, why not return `PathBuf` directly in `default_data_path`?

Reply:
Fixed